regen-network/regen-web
# chore: update profile dropdown

## Description

Closes: regen-network/rnd-dev-team#1875



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2304--regen-marketplace.netlify.app/
1. Login and check the new profile dropdown UI
2. Log in with Keplr then switch account from Keplr. You should see a warning modal in the app and the Keplr "prove ownership" popup
   1. If you approve then you're logged out from the previous account and logged in with the new one
   2. If you reject, then you'll still be logged in with the previous account but the wallet address won't be the same as the one from your current account so as soon as you want to perform an on chain action, you'll get a modal saying to switch to the right account in Keplr
   3. If you switch back to the initial account from Keplr, then you are still logged in with the initial account

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
